README.md

```diff
@@ -127,8 +127,8 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
         .build();
 
     // Start up the cluster
-    let cluster = Cluster::new(config);
-    cluster.up().await?;
+    let cluster = Cluster::new(config).await?;
+    cluster.up().await;
```

Comment:
Should this be changed to be in a task as well?

Reply:
Sure!

Reply:
Done :)